File: IOT/shakap12_ShashwatKapoor_1923ec1177_3/Patterns/Pattern10.java
import java.util.*;

public class Main{

public static void main(String[] args) {
    Scanner scn = new Scanner(System.in);

     // write ur code here
     int n=scn.nextInt();
     int spacesf=n/2,spacesb=-1;
     for(int i=0;i<n;i++){
         for(int j=0;j<spacesf;j++){
             System.out.print("\t");
         }
         System.out.print("*"+"\t");
         for(int j=0;j<spacesb;j++){
             System.out.print("\t");
         }
         if(i!=0 && i!=n-1){
             System.out.print("*");
         }
         if(i<n/2){
             spacesf--;
             spacesb+=2;
         }
         else if(i>=n/2){
             spacesf++;
             spacesb-=2;
         }
         System.out.println();
     }

 }
}

File: Web/Khushi-Vaish_KhushiVaish_2024cs1147_2/Patterns/pattern10.cpp
import java.util.*;

public class Main{

public static void main(String[] args) {
    Scanner scn = new Scanner(System.in);
    int n = scn.nextInt();
    
    int p = n / 2;
    int q = -1;
    for(int i = 1; i <= n; i++){
       for(int j = 1; j <= p; j++){
        System.out.print("\t");
       }
       System.out.print("*\t");

       for(int j = 1; j <= q; j++){
        System.out.print("\t");
       }
       if(i > 1 && i < n){
        System.out.print("*\t");
       }
       
       if(i <= n / 2){
        p--;
        q += 2;
       } else {
        p++;
        q -= 2;
       }

        System.out.println();
    }

 }
}


 